### PR TITLE
Adding Redirect for Leadership calendar

### PR DESCRIPTION
**Pull request by sebworks**
Adding Redirect for Leadership calendar

## Changes

- Modified `cfgov/cfgov/urls.py` to adding redirect to Wagtail page.
- Modified `cfgov/v1/util/util.py` to add secondary navigation items.

## Testing

- Download the build dump
- Change the slug of the `CFGov -> About-Us -> Leadership Calendar` page to `leadership-calendar`.
- Visit `http://localhost:8000/about-us/the-bureau/leadership-calendar/` and `http://localhost:8000/about-us/leadership-calendar/`. Both URLs should render the Wagtail Leadership page.

## Notes

-

## Todos

-

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

**Comment by richaagarwal** [comment]
@sebworks Could you provide more context for these changes?  If the page exists in Wagtail, why does it need all this custom rendering?

**Comment by sebworks** [comment]
The rest of the pages are Sheer pages. GHE issue 370

**Comment by schaferjh** [comment]
Thanks @sebworks. Do we need to do work to take down the current Leadership Calendar page before this redirect goes up?

**Comment by sebworks** [comment]
@schaferjh, no the content team will just need to publish the page `'about-us/leadership-calendar'` once this code is deployed to production. We will need to create a task to delete the old Leadership Calendar Sheer code.

**Comment by schaferjh** [comment]
👍

File: cfgov/v1/util/util.py [hunk 1]
```diff
@@ -65,6 +65,33 @@ def get_secondary_nav_items(request, current_page):
         ], True
     # END TODO
 
+    # TODO: Remove this ASAP once the-bureau gets migrated to Wagtail
```
Comment (richaagarwal) [inline cfgov/v1/util/util.py]: Can we perhaps create a task for this so we don't forget to do it?

**Comment by richaagarwal** [comment]
Can you update the CHANGELOG? :)

**Pull request merged**